pkg/watch/json/types.go

```diff
@@ -32,7 +32,7 @@ import (
 type WatchEvent struct {
 	// The type of the watch event; added, modified, deleted, or error.
 	// +optional
-	Type watch.EventType `json:"type,omitempty" description:"the type of watch event; may be ADDED, MODIFIED, DELETED, or ERROR"`
+	Type watch.EventType `json:"type,omitempty" description:"the type of watch event; may be ADDED, MODIFIED, DELETED, BOOKMARK or ERROR"`
```

Comment:
#74074 will be merged first?

Reply:
As mentioned offline - we have a policy to not merge API changes before we have an implementation...
So yet - this will be merged first, but we need to have this one working to merge that.

Reply:
done